decentraland/unity-renderer
# fix: biw bug fixing

## What this PR includes?

- Some entities with two renderers sometimes doesn't load the second one making the tool unusable

- Focus on missing objects moves the camera far away

- The camera should be frame rate - independent

- The camera is glitching a little if you just use the right click on the mouse

## How to test it?


1. Go to: https://play.decentraland.zone/index.html?realm=artemis-amber&renderer=urn%3Adecentraland%3Aoff-chain%3Arenderer-artifacts%3Afix%2Fbiw-bug-fixing-3
2. Enter builder in world
3. Check that the fixes are apply

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
